src/components/SecondaryPanes.js

```diff
@@ -45,6 +51,39 @@ const SecondaryPanes = React.createClass({
 
   displayName: "SecondaryPanes",
 
+  renderBreakpointsToggle() {
+    const { toggleAllBreakpoints, breakpoints,
+            breakpointsDisabled, breakpointsLoading } = this.props;
+    const boxClassName = "breakpoints-toggle";
+
+    if (breakpoints.size == 0 || breakpointsLoading) {
+      return dom.input({
+        type: "checkbox",
+        className: boxClassName,
+        disabled: true,
+        title: L10N.getStr("breakpoints.disable")
+      });
+    }
+
+    return dom.input({
+      type: "checkbox",
+      className: boxClassName,
+      onClick: () => toggleAllBreakpoints(!breakpointsDisabled),
+      defaultChecked: true,
+      ref: (input) => {
+        if (input != null) {
+          const isIndeterminate = !breakpointsDisabled &&
+            breakpoints.some(x => x.disabled);
+          input.indeterminate = isIndeterminate;
+          // checked=true would override the indeterminate value
+          input.checked = !breakpointsDisabled && !isIndeterminate;
```

Comment:
Was going to say the same thing  :) They both are standard dom properties, so it should just work as normal props